cli/src/config.rs

```diff
@@ -186,6 +180,15 @@ pub fn get_home_settings_path() -> Result<PathBuf> {
     Ok(config_path)
 }
 
+/// Resolve XDG config directory.
+pub fn config_dir(home_path: &Path) -> PathBuf {
+    env::var("XDG_CONFIG_HOME")
```

Comment:
I've noticed the previous version incorrectly uses `XDG_CONFIG_DIR`, which isn't a thing. I've separated the fix for that in a separate commit so I'd recommend rebasing this PR rather than squashing so that fix is preserved as a separate commit.